metaworld/envs/mujoco/sawyer_xyz/sawyer_push_wall_v2.py

```diff
@@ -0,0 +1,201 @@
+"""Version 2 of SawyerPushWallEnv."""
+
+import numpy as np
+from gym.spaces import Box
+from metaworld.envs.env_util import get_asset_full_path
+from metaworld.envs.mujoco.sawyer_xyz.base import SawyerXYZEnv
+
+
+class SawyerPushWallEnvV2(SawyerXYZEnv):
+    """
+    SawyerPushEnvV2 updates SawyerReachPushPickPlaceWallEnv.
+
+    Env now handles only 'Push' task type from SawyerReachPushPickPlaceWallEnv.
+    Observations now include a vector pointing from the objectposition to the
+    goal position. Allows for scripted policy.
+    """
```

Comment:
I feel like we should keep the structure of these comments the same across V2 envs. At least add a "Changelog" header and list the changes under it. It's easier to read than paragraph form.